atomic_reactor/plugins/exit_remove_worker_metadata.py

```diff
@@ -27,7 +27,11 @@ def run(self):
         """
         build_result = self.workflow.build_result
 
-        worker_builds = build_result.annotations['worker-builds']
+        if not build_result.annotations:
+            self.log.info("No build annotations found, skipping plugin")
```

Comment:
BTW, what are build annotations?

Reply:
Openshift annotations, we are storing few build metadata there and then fetch it from orchestrator.